src/utils/tax-year-to-text.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (question) {
+  const data = {};
+
+  if (question?.info?.currentTaxYear) {
+    const currentTaxYear = question.info.currentTaxYear;
+    const [startYear] = currentTaxYear.split("/");
+    const formattedYear = `${startYear} to ${parseInt(startYear, 10) + 1}`;
```

Comment:
Just had a thought, will the "to" here get translated for Welsh? otherwise we might need to rework this